examples/common/avb.c

```diff
@@ -72,10 +72,17 @@ int pci_connect(device_t * igb_dev)
 
 		printf("attaching to %s\n", devpath);
 		err = igb_attach(devpath, igb_dev);
-		if (err) {
+		if (err || igb_attach_tx(igb_dev)) {
```

Comment:
Yes, without that igb_xmit fails. We should probably have all example applications use these calls from common/avb.c. I'm guessing the common directory was created for this specific reason. At the moment it's there in multiple places
- simple_talker.c
- jackd_talker.c
- common/avb.c

@pinealservo simple_talker.c does this _if (err || igb_attach_tx())_. May be that needs to be fixed too. I got the simple talker/listener applications working first and then moved on to this one.